manifests/d/Datalust/Seq/2021.2.6288/Datalust.Seq.yaml

```
@@ -0,0 +1,34 @@
+PackageIdentifier: Datalust.Seq
+PackageVersion: 2021.2.6288
+PackageName: Seq
+PackageLocale: en-US
+Publisher: Datalust
+PackageUrl: https://datalust.co/seq
+PublisherUrl: https://datalust.co
+PublisherSupportUrl: https://discuss.datalust.co
+License: Copyright © 2013-2021 Datalust
+LicenseUrl: https://datalust.co/doc/eula-current.pdf
+MinimumOSVersion: 10.0.0.0
+ShortDescription: >-
+  Seq is the easiest way for development teams to capture, search and visualize structured
+  log events.
+Tags:
+  - Logging
+  - Developer-Tools
```

Best practice is to use all lower case. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

> The best practice is to present these terms in all lower case with hyphens rather than spaces.

```suggestion
Tags:
  - logging
  - developer-tools
```